### PR TITLE
Add a simple readme

### PR DESCRIPTION
**Pull request by plckr**
Hope it'll be helpful at some point, isn't much detailed but it's better than nothing, right.
Let me know if you want something more specific to be added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+```js
+import { useTranslation } from '@significa/react-i18n'
+
+const YourComponent = () => {
+  const { t } = useTranslation()
+
+  return (
+    <div>
+      <p>{t.yourMessage}</p>
```
Comment (plckr) [inline README.md]: this is a typo: should be `t('yourMessage')`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+```
+{
+  id: message,
+  anotherId: anotherMessage
+}
+```
+
+You have to deal the language switcher outside this package.
+If using Next.js you'll have that out of the box
```
Comment (pdrbrnd) [inline README.md]: not sure what you mean by this.
maybe we should move this paragraph next to the Provider and explain that we should only load the current language's strings to the provider?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Message parameters
+
+It's possible to add parameters to the message, a string between double curlybraces is represented as a parameter.
+
+
+```js
+// Messages Object
+{
+  term: "You searched for {{term}}"
```
Comment (pdrbrnd) [inline README.md]:
```suggestion
  greet: "Hello {{name}}"
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+}
+```
+
+```js
+<Translation id="blog.search" values={{
+  term: searchedTerm   // {{term}} will be replaced for searchedTerm
+}} />
```
Comment (pdrbrnd) [inline README.md]:
```suggestion
<Translation id="greet" values={{
  name: user.name   // {{name}} will be replaced by `user.name`
}} />
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```
```
Comment (pdrbrnd) [inline README.md:94]: maybe we should add an example with the normal `t` syntax as well?

**Review by pdrbrnd — CHANGES_REQUESTED** [review]
@plckr thanks a lot for this! it's great and very much appreciated.
I just left some small comments. Can you take a look at it?

File: README.md (hunk 2 above)
Comment (plckr) [reply README.md]: @pdrbrnd I think it's more clear now, let me know what you think

**Review by pdrbrnd — APPROVED** [review]

**Pull request merged**